nuts_and_bolts/main/routes.py

```diff
@@ -66,3 +66,12 @@ def add_to_cart(id):
     else:
         flash('Unable to add to cart. That product does not exist.', 'danger')
     return redirect(url_for('main.show_cart'))
+
+@main.route('/clear_cart')
+def clear_cart():
+        session.clear()
```

Comment:
This should be "session['cart'] = {}" instead of nuking the entire session. Clearing the session will cause errors when trying to access the cart elsewhere as it will no longer exist.